code/modules/mob/living/carbon/human/death.dm

```diff
@@ -13,9 +13,9 @@
 
 		playsound(src.loc, 'sound/effects/gib.ogg', 100, 1, 10)
 
-	for(var/obj/item/organ/I in internal_organs)
+	for(var/obj/item/organ/internal/I in internal_organs)
 		if(istype(loc,/turf))
-			I.removed()
+			I.remove(src)

```

Comment:
I'm pretty sure removed doesn't set the location, so this will just throw items that are in null space to....null space.